src/message.rs

```diff
@@ -1355,6 +1355,14 @@ pub async fn get_msg_info(context: &Context, msg_id: MsgId) -> Result<String> {
             ret += &format!("\nLast seen as: {}/{}", server_folder, msg.server_uid);
         }
     }
```

Comment:
I don't know how to fix the clippy-issue originating from these lines. I tried replacing it by a map, but then I get: 
error: called `map(f)` on an `Option` value where `f` is a closure that returns the unit type `()`